src/styles/blocks/catalog.scss

```
@@ -17,9 +17,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-property: transform, color;
+    transition-duration: 300ms, 300ms;
```

```suggestion
    transition: transform 300ms, color 300ms;
```
And move 300ms to the variable